core/src/App.html

```diff
@@ -1348,10 +1348,11 @@
 
       if ('luigi.navigation.open' === e.data.msg) {
         isNavigateBack = false;
-
         const srcNode = isSpecialIframe ? iframe.luigi.currentNode : undefined;
         const srcPathParams = isSpecialIframe ? iframe.luigi.pathParams : undefined;
-
+        if (e.source !== window && !e.data.params.intent && e.data.params.link) {
+          e.data.params.link = e.data.params.link.split('?')[0];
+        }
```

Comment:
```suggestion
        const params = e.data.params;
        const { intent, link, newTab, modal, splitView, drawer, withoutSync } = params;
        if (e.source !== window && !intent && link) {
          link = link.split('?')[0];
        }
```

We repeated a lot of `e.data.params` here, as well as inside `luigi.navigation.open`.

Reply:
We could declare `e.data.params` on the top and make the other conditions(newTab, drawer, splitView, etc...) shortly.